2021/viz_evaluation.md

```diff
@@ -0,0 +1,70 @@
+# Evaluation of data visualization effectiveness
+ArviZ is a Python and Julia library for exploratory analysis of Bayesian models,
+and includes many [vizualizations](https://arviz-devs.github.io/arviz/examples/index.html)
+to that end.
+
+As ArviZ developers we try to follow the literature on effective data visualization,
+especially around communicating uncertainty, but we might not have time to
+keep up with everything, we might be weighted down by bad practices we still have
+not managed and we might not always have time to update the code when a new
+recommendation or visualization type is made available.
+Probably a combination of all 3 and some extra factors.
+
+Therefore, to ensure ArviZ is an active agent in advocating and popularizing
+effective data visualization practices we budgeted $2000 as part of our GSoD
+2021 application to pay someone external to the team to evaluate the
+plots that are available in ArviZ and their defaults.
+
+## Job description
+We are expecting a report covering the following points:
+
+* Analysis of available and missing visualizations
+
+  Evaluate which visualizations should be present in ArviZ but are not.
+  Each visualization list how it would serve ArviZ's goal and some
+  degree of addition priority, not necessarily a ranking, could be
+  binning missing visualizations in 3-4 high-low priority categories.
```

Comment:
```suggestion
  For each visualization we should get a description of how it would serve ArviZ's goal and some degree of addition priority, not necessarily a ranking, could be
  binning missing visualizations in 3 or 4 high to low priority categories.
```